backend/cognito/template.yaml

```diff
@@ -0,0 +1,494 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  Cognito UserPool and Client
+Transform:
+  - AWS::Serverless-2016-10-31
+
+Resources:
+  UserPool:
+    Type: AWS::Cognito::UserPool
+    Properties:
+      Policies:
+        PasswordPolicy:
+          MinimumLength: 6
+          RequireUppercase: false
+          RequireLowercase: false
+          RequireNumbers: false
+          RequireSymbols: false
+          TemporaryPasswordValidityDays: 7
```

Comment:
Main reason is much of this is the output of `aws cognito-idp describe-user-pool id` passed through yq.

But, we are being explicit about our password constraints (which is just length; it's set to 6 because we need that for the six-digit email code but if we come up with a minimum length we'll have to enforce that in Express middleware).

The documentation isn't clear on what the default value is if these are omitted which is another reason to be explicit about it.

I don't think we gain anything by omitting these and hoping the default is always false.